src/reference/04-Howto/13-Howto-Custom-Tasks.md

```diff
@@ -0,0 +1,15 @@
+Define Custom Tasks 
+-------------------
+
+### Define a Task that runs tests in specific sub-projects
+
+Defines a task `myTestTask` that will run the `test` Task in specific subprojects  `abc` and `def` but no others:
```

Comment:
`def` is a keyword, so it's probably not a good idea to use it here. The current recommendation is to use multi-project build.sbt style. Also wouldn't it be a better example if it's self-complete? i.e. define the subprojects it's referring to?